packages/account/src/Components/account-limits/account-limits.jsx

```diff
@@ -128,7 +128,7 @@ const AccountLimits = ({
                                 <thead>
                                     <tr>
                                         <AccountLimitsTableHeader>
-                                            <Localize i18n_default_text='Trading limits - Item' />
+                                            <Localize i18n_default_text='Trading limits' />
```

Comment:
The test cases  still have the title "/trading limits \- item".  can you double check the test cases